elements/drag-drop-elements/pl-drag-drop-element.py

```diff
@@ -0,0 +1,369 @@
+import prairielearn as pl
+import lxml.html
+import random
+import chevron
+import base64
+import os
+import json
+
+# Read https://prairielearn.readthedocs.io/en/latest/devElements/
+# Official documentation on making custom PL
+
+
+def render_html_colour(score):
+    # used to render the correct colour depending on student score
+    if score == 0:
+        return 'badge-danger'
+    elif score == 1.0:
+        return 'badge-success'
+    else:
+        return 'badge-warning'
+
+# courtesy of https://github.com/PrairieLearn/PrairieLearn/pull/2625
+
+
+def getallAnswer(submitted_blocks, block_indents, leading_code, trailing_code):
```

Comment:
some of the function and variable naming in this file doesn't follow the PEP8 style guide (https://www.python.org/dev/peps/pep-0008/) 
as it should.

I know its a bit confusing that the convention is different for modules vs. classes vs. functions....we'll just have to blame the Python committee for that.

Reply:
corrected

Reply:
there are some inconsistencies in local variables too. Most are snake case, but you have a few camel case like `answerName`, `answerNameField`, `permutationMode`, `isShuffle`. 

That's the only other problem I can see right now.

Reply:
ooo ok, ill get on that